vpr/src/place/place.cpp

```diff
@@ -1491,8 +1541,8 @@ static e_move_result try_swap(float t,
             /* Update clb data structures since we kept the move. */
             commit_move_blocks(blocks_affected);
 
-        } else { /* Move was rejected.  */
-                 /* Reset the net cost function flags first. */
+        } else if ((!manual_move && move_outcome == REJECTED) || (manual_move && manual_move_outcome == REJECTED)) { /* Move was rejected.  */
```

Comment:
Simplify by having one move_outcome state for all moves.

Reply:
resolved